src/main/resources/messages/LibertyBundles.properties

```
@@ -32,34 +32,34 @@ action.io.openliberty.tools.intellij.actions.ViewGradleConfig.description=Open t
 action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.text=Liberty: Refresh tool window
 action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.description=Refresh tool window Liberty projects
 action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.text=Liberty: Run Liberty action
-action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.description=Run the selected Liberty action
+action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.description=Run the selected Liberty action.
 action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.text=Liberty: Add project to the tool window
 action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.description=Add the selected Liberty project to the tool window
 action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.text=Liberty: Remove project from the tool window
 action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.description=Remove the selected Liberty project from the tool window
 
 # Messages for Liberty actions and pop-up dialog
-no.liberty.projects.detected=No Liberty Maven or Liberty Gradle projects detected in this workspace.
-liberty.dev.not.started.notification.content=Unable to {0}. Liberty dev mode may not have been started on {1}. {2}Use the Liberty tool window to start Liberty dev mode.
-liberty.project.does.not.resolve=Unable to {0}: could not resolve project. Ensure you run the Liberty action from the Liberty tool window.
-liberty.build.file.does.not.resolve=Unable to {0}: could not resolve build file for {1}. Ensure you run the Liberty action from the Liberty tool window.
+no.liberty.projects.detected=No Liberty Maven or Liberty Gradle projects are detected in this workspace.
+liberty.dev.not.started.notification.content=Unable to {0}. Liberty dev mode might not be started on {1}. {2} Use the Liberty tool window to start Liberty dev mode.
```

The third parameter (`{2}`) for this message is actually a line separator, so we should leave no space.
https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java#L174
```suggestion
liberty.dev.not.started.notification.content=Unable to {0}. Liberty dev mode might not be started on {1}. {2}Use the Liberty tool window to start Liberty dev mode.
```